### ISSUE TITLE
As of v0.9.0 dbt_utils.except() macro is now deprecated and showing a warning

### ISSUE THREAD

**Issue by daveworth** [EqualExperts/dbt-unit-testing#88]
The dbt_utils package has been working to pull many of its macros into dbt-core rather than the utils package.  This was done with [`dbt_utils.except()`](https://github.com/dbt-labs/dbt-utils/blob/0.9.1/macros/cross_db_utils/deprecated/except.sql#L6) which means tests from `dbt-unit-testing` are now raising deprecation warnings (though tests are still running as expected):

```
Warning: the `except` macro is now provided in dbt Core. It is no longer available in dbt_utils and backwards compatibility will be removed in a future version of the package. Use `except` (no prefix) instead. The <some-project>-<some-model> model triggered this warning.
```

**Comment by the-serious-programmer** [comment]
Our log output is also getting spammed by this, would love to see this resolved!

**Comment by cdiniz** [comment]
Hi @daveworth and @the-serious-programmer, thanks for the feedback.
Solve by #94 
Available in [v0.2.4](https://github.com/EqualExperts/dbt-unit-testing/releases/tag/v0.2.4)

**Comment by daveworth** [comment]
Thank you @cdiniz - I appreciate the fix!
